services/ui-src/src/routes/Dashboard/Dashboard.test.tsx

```diff
@@ -7,11 +7,13 @@ import { Dashboard } from "routes";
 import { ReportContext } from "components";
 // utils
 import {
-  mockAdminUser,
-  mockHelpDeskUser,
+  /*
```

Comment:
I commented out the code in this file so @TheDanMiller has it for reference when implementing archive functionality. Just thought it would be more helpful than throwing it away since he'll be doing this so soon.